lib/ecto/pools/sojourn_broker/worker.ex

```diff
@@ -41,6 +41,9 @@ defmodule Ecto.Pools.SojournBroker.Worker do
     backoff_threshold = div(max_backoff, 3)
     lazy = Keyword.get(worker_opts, :lazy, true)
 
+    # Seed random number generator for backoff
+    :random.seed(:erlang.now)
```

Comment:
As an aside: The actual implementation in rand.erl doesn't use `erlang:phash2([node()])` it uses `erlang:phash2([{node(),self()}])`.

https://github.com/erlang/otp/blob/maint/lib/stdlib/src/rand.erl#L88-L90